tests/unit/orcid/fixtures/works.json

```diff
@@ -0,0 +1,141 @@
+                        },
+                        "day": null,
+                        "month": null
+                    },
+                    "type": "JOURNAL_ARTICLE"
+                }
+            ],
+            "last-modified-date": {
+                "value": 1516722056845
+            }
+        },
+        {
+            "external-ids": {
+                "external-id": [
+                    {
+                        "external-id-value": "astro-ph/0002520",
+                        "external-id-relationship": "SELF",
+                        "external-id-type": "arxiv",
+                        "external-id-url": {
+                            "value": "http://arxiv.org/abs/astro-ph/0002520"
+                        }
+                    }
+                ]
+            },
+            "work-summary": [
+                {
+                    "last-modified-date": {
+                        "value": 1516716538439
+                    },
+                    "created-date": {
+                        "value": 1516715359801
+                    },
+                    "visibility": "PUBLIC",
+                    "display-index": "0",
+                    "path": "/0000-0001-7102-4649/work/912977",
+                    "put-code": 912977,
+                    "title": {
+                        "title": {
+                            "value": "CMB anisotropies: A Decadal survey"
+                        },
+                        "subtitle": null,
+                        "translated-title": null
+                    },
+                    "external-ids": {
+                        "external-id": [
+                            {
+                                "external-id-value": "astro-ph/0002520",
+                                "external-id-relationship": "SELF",
+                                "external-id-type": "arxiv",
+                                "external-id-url": {
+                                    "value": "http://arxiv.org/abs/astro-ph/0002520"
+                                }
+                            }
+                        ]
+                    },
+                    "source": {
+                        "source-orcid": null,
+                        "source-name": {
+                            "value": "INSPIRE-PROFILE-PUSH"
+                        },
+                        "source-client-id": {
+                            "path": "0000-0002-0970-6486",
+                            "host": "sandbox.orcid.org",
+                            "uri": "http://sandbox.orcid.org/client/0000-0002-0970-6486"
+                        }
+                    },
+                    "publication-date": null,
+                    "type": "CONFERENCE_PAPER"
+                }
+            ],
+            "last-modified-date": {
+                "value": 1516716538439
+            }
+        }
+    ],
+    "last-modified-date": {
+        "value": 1516722056845
+    }
+}
```

Comment:
I would suggest a more explicit name for the fixture.

Reply:
Done :+1: